app/code/Magento/Ui/Config/Data.php

```diff
@@ -83,7 +84,8 @@ public function __construct(
         $this->serializer = $serializer;
         $this->componentName = $componentName;
         $this->argumentInterpreter = $argumentInterpreter;
-        $this->cacheId = static::CACHE_ID . '_' . $componentName;
+        $storeManager = \Magento\Framework\App\ObjectManager::getInstance()->get(StoreManagerInterface::class);
```

Comment:
Still must be passed via constructor. Just don't assign to property.

Reply:
Fixed, please check.